src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Utilities/Converters.cs

```diff
@@ -0,0 +1,341 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using static System.Diagnostics.Debug;
+
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Utilities
+{
+    [ValueConversion(typeof(bool), typeof(Visibility))]
+    internal class InvertableBooleanToVisibilityConverter : IValueConverter
```

Comment:
Rather than putting all these UI helpers (converters.cs, delegatecommand.cs, etc) under Utilities, they should probably be moved to UI.